public/app/percona/backup/components/BackupInventory/RestoreBackupModal/RestoreBackupModal.styles.ts

```diff
@@ -0,0 +1,49 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = ({ colors, typography, spacing }: GrafanaTheme) => ({
+  label: css`
+    display: block;
+    text-align: left;
+    font-size: ${typography.size.md};
+    font-weight: ${typography.weight.semibold};
+    line-height: 1.25;
+    margin: ${spacing.formLabelMargin};
+    padding: ${spacing.formLabelPadding};
+    color: ${colors.formLabel};
+  `,
+  select: css`
+    margin-bottom: ${spacing.xl};
+    div[class$='-input-wrapper'] {
+      padding: 7px 8px;
```

Comment:
let's try to stick to the `spacing` values from the theme (same few lines below)

Reply:
Same reason as above